src/eventEmitter.ts

```diff
@@ -65,6 +65,27 @@ export const purchaseUpdatedListener = (
   return emitterSubscription;
 };
 
+export const userChoiceBillingUpdateListener = (
```

Comment:
Maybe `userChoiceBillingUpdateListener` to `userChoiceBillingUpdateListenerAndroid`?